cmd/av/stack_submit.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"context"
+
+	"emperror.dev/errors"
+	"github.com/aviator-co/av/internal/actions"
+	"github.com/aviator-co/av/internal/config"
+	"github.com/aviator-co/av/internal/gh"
+	"github.com/aviator-co/av/internal/meta"
+	"github.com/shurcooL/githubv4"
+	"github.com/spf13/cobra"
+)
+
+var stackSubmitCmd = &cobra.Command{
+	Use:   "submit",
+	Short: "create/synchronize pull requests for the current stack",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) > 0 {
+			_ = cmd.Usage()
+			return errors.New("this command takes no arguments")
+		}
+
+		// Get the all branches in the stack
+		repo, _, err := getRepoInfo()
+		if err != nil {
+			return err
+		}
+		branches, err := meta.ReadAllBranches(repo)
+		if err != nil {
+			return err
+		}
+		currentBranch, err := repo.CurrentBranchName()
+		if err != nil {
+			return err
+		}
+		previousBranches, err := meta.PreviousBranches(branches, currentBranch)
+		if err != nil {
+			return err
+		}
+		subsequentBranches, err := meta.SubsequentBranches(branches, currentBranch)
+		if err != nil {
+			return err
+		}
+		var branchesToSubmit []string
+		branchesToSubmit = append(branchesToSubmit, previousBranches...)
+		branchesToSubmit = append(branchesToSubmit, currentBranch)
+		branchesToSubmit = append(branchesToSubmit, subsequentBranches...)
+
+		// ensure pull requests for each branch in the stack
+		ctx := context.Background()
+		client, err := gh.NewClient(config.Av.GitHub.Token)
+		if err != nil {
+			return err
+		}
+		for _, branchName := range branchesToSubmit {
+			result, err := actions.CreatePullRequest(
+				ctx, repo, client,
+				actions.CreatePullRequestOpts{
+					BranchName: branchName,
+					Draft:      config.Av.PullRequest.Draft,
+				},
+			)
+			if err != nil {
+				return err
+			}
+			// make sure the base branch of the PR is up to date if it already exists
+			if !result.Created {
```

Comment:
Can we make this only happen if `BaseRefName` is wrong? Also, would be nice to make sure we add `avbeta-stackedprs` label here if it's not set (that should probably be a followup PR though).